tsc/process/release.md

```diff
@@ -0,0 +1,20 @@
+**Release Process for OpenVDB**
+
+The following assumes that the current OpenVDB library version number is 6.0.0 and the new version number is 6.1.0. Adjust for the actual version numbers as appropriate.
+
+- [ ] Open a Jira "Release OpenVDB 6.1.0" ticket with "OpenVDB_6.1.0" as the Fix Version.
```

Comment:
Note that if we use the JIRA ASWF for this, we'll need to adjust the configuration to add "Fix Version" (among other currently disabled features referenced here). I'm fine with leaving this as-is for the time being though.

Reply:
Right, and at least one of us would have to have the necessary permissions to create new versions.